packages/web3-wrapper/src/index.ts

```diff
@@ -4,6 +4,8 @@ import * as _ from 'lodash';
 import * as Web3 from 'web3';
 
 export class Web3Wrapper {
+    // This is here purely to reliably distinguish it from other objects in runtime (like BigNumber.isBigNumber)
+    public isZeroExWeb3Wrapper = true;
```

Comment:
BigNumber removed `isBigNumber` in the latest version, and instead use `typeof`. Why can't we use `typeof`?

Reply:
Yep, brainfart, carry on.